internal/tools/securitypolicy/README.md

```diff
@@ -94,6 +104,37 @@ represented in JSON.
             "0": "/path/to/container/mount-1",
             "1": "/path/to/container/mount-2"
           }
+        },
+        "mounts": {
```

Comment:
Is there a description anywhere that discusses the differences between an "expected_mount" and a "mount"?

Reply:
I think it's the case of bad naming... `expected_mount` should probably be `wait_paths` or `wait_mount_points` instead. The current name suggests that there's some sort of enforcement mechanism for the mounts in `expected_mount`, where in reality it's just a list of paths that should be present when container starts.

Reply:
is that something we could easily change later or are we locked into the naming?

Reply:
we can change it later, it's not a problem.

Reply:
My vote would be changing it after this PR is merged or as soon as is convenient

Reply:
sounds good. we have a thread with tooling team to finalize on the naming here.